src/style.css

```
@@ -0,0 +1,80 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.flex-container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  min-width: 300px;
+  font-size: 100px;
+  color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-basis: 33.33%;
+  flex-grow: 1;
+}
+
+.block--1 {
+  background-color: red;
+}
+
+.block--2 {
+  background-color: rgb(80%, 0%, 0%);
```

Stick with the same color definitions
```suggestion
  background-color: rgb(100%, 0%, 0%);
}

.block--2 {
  background-color: rgb(80%, 0%, 0%);
```